modules/odc-deploying-java-applications.adoc

```diff
@@ -2,40 +2,42 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
```

Comment:
```suggestion
= Uploading JAR files for easy deployment of Java applications
```
Because this is a procedural module, the title should begin with a gerund phrase.
Reference: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs